Library/Homebrew/manpages/brew.1.md

```diff
@@ -345,12 +345,30 @@ Note that these flags should only appear after a command.
   * `switch` <name> <version>:
     Symlink all of the specific <version> of <name>'s install to Homebrew prefix.
 
-  * `tap` [--full] [<tap>]:
-    Tap a new formula repository from GitHub, or list existing taps.
-
-    <tap> is of the form <user>/<repo>, e.g. `brew tap homebrew/dupes`.
-
-    If `--full` is passed, a full clone will be used.
+  * `tap` [--full] [<name>] [<target>]:
+    Tap a new formula repository or list existing taps. This command can be
+    invoked in three ways. In what follows, <name> stands for something of the
+    form <user>/<repo>, e.g. `brew tap homebrew/dupes`.
+
+    + `tap` without arguments displays existing taps.
+    + `tap <name>` taps a new formula repository from GitHub using HTTP
```

Comment:
I went back and forth about this, but picked HTTP because I was thinking of the protocol more than the specifics (i.e. HTTP rather than SSH). But I'll change it.